apps/web/src/app/(recipient)/d/[token]/sign-direct-template.tsx

```diff
@@ -169,6 +169,55 @@ export const SignDirectTemplateForm = ({
     // Do not reset to false since we do a redirect.
   };
 
+  useEffect(() => {
```

Comment:
this signs the non-radio/non-checkboxes advanced fields if they have a default value. without this useEffect, they don't automatically sign if the document contains checkboxes/radio fields.

maybe I'm missing something.